src/dev/code_coverage/shell_scripts/merge_jest_and_functional.sh

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
 
-EXTRACT_START_DIR=tmp/extracted_coverage
-EXTRACT_END_DIR=target/kibana-coverage
-COMBINED_EXTRACT_DIR=/${EXTRACT_START_DIR}/${EXTRACT_END_DIR}
+COVERAGE_TEMP_DIR=/tmp/extracted_coverage/target/kibana-coverage/
 
 echo "### Merge coverage reports"
 for x in jest functional; do
-  yarn nyc report --temp-dir $COMBINED_EXTRACT_DIR/${x} --report-dir $EXTRACT_END_DIR/${x}-combined --reporter=html --reporter=json-summary
+  yarn nyc report --nycrc-path src/dev/code_coverage/nyc_config/nyc.${x}.config.js
```

Comment:
@wayneseymour maybe you have an idea why it can fail with
```
17:58:16  ### Merge coverage reports
17:58:16  yarn run v1.22.4
17:58:16  $ /var/lib/jenkins/workspace/elastic+kibana+coverage-ingestion/kibana/node_modules/.bin/nyc report --nycrc-path src/dev/code_coverage/nyc_config/nyc.jest.config.js
17:58:16  ENOENT: no such file or directory, scandir '/var/lib/jenkins/workspace/elastic+kibana+coverage-ingestion/kibana/target/kibana-coverage/jest'
17:58:16  error Command failed with exit code 1.
```

Reply:
Maybe we can list the contents of all the paths in question?

Reply:
The issue was caused by mixing path during `sed` operation. I put it out of this PR, we anyway need to think about the smart way of replacing path in jsons

Reply:
it is working fine now